src/scss/includes/direction.scss

```diff
@@ -977,9 +975,19 @@ button.direction_shortcut {
     color: #353C52;
   }
 
-  input:focus ~ .itinerary_field_icon {
-    right: 42px;
-    top: 12px;
+  input:focus {
+    // border: 1px solid grey;
+    // border-radius: 50px;
+
+    &::before {
+      content: '';
+      position: fixed;
+      top: 0; right: 0; bottom: 0; left: 0;
+      margin: -3px;
+      border-radius: inherit;
+      z-index: 1;
+      background: $focus-gradient;
+    }
```

Comment:
@G-Ray Did you mean to commit this change ? The gradient is not visible on mobile on my side.

Reply:
I did not want to, sorry I remove it rn